processing/src/main/java/io/druid/query/aggregation/BufferAggregator.java

```diff
@@ -178,4 +178,26 @@ default void relocate(int oldPosition, int newPosition, ByteBuffer oldBuffer, By
   {
   }
 
+  /**
+   * Returns true if the aggregator is nullable and the aggregated value is null
```

Comment:
Same as in Aggregator

Reply:
done.

Reply:
Please update this doc according to the new policy